### DISCUSSION TITLE
Configuration options

### DISCUSSION THREAD

**Discussion by pjanzen** [Netflix/conductor#3160] [General]
Hi All,

I am looking from the configration options for the different backends you can configure. I have managed to find the config-postgres.properties file in de docker but I am sure there is some documentation that I am missing.

I hope someone can point me in the right direction.

Regards,
Paul.

**Comment by ZergRushJoe** [comment]
```
    # Servers.
    conductor.grpc-server.enabled=false

    # Database persistence type.
    conductor.db.type=postgres

    spring.datasource.url=jdbc:postgresql://{{ .Release.Name }}-db.{{ .Release.Namespace }}.svc.cluster.local:{{ .Values.services.database.exposedPort }}/conductor
    spring.datasource.username={{ .Values.db.username }}
    spring.datasource.password={{ .Values.db.password }}

    # Hikari pool sizes are -1 by default and prevent startup
    spring.datasource.hikari.maximum-pool-size=10
    spring.datasource.hikari.minimum-idle=2

    # Elastic search instance indexing is enabled.
    conductor.indexing.enabled=true

    # Transport address to elasticsearch
    conductor.elasticsearch.url=http://{{ .Release.Name }}-elasticsearch.{{ .Release.Namespace }}.svc.cluster.local:{{ .Values.services.cache.exposedPort }}

    # Name of the elasticsearch cluster
    conductor.elasticsearch.indexName=conductor

    # Yellow is main cluster node is up and running for elasticsearch
    conductor.elasticsearch.clusterHealthColor=yellow
```
@pjanzen here is a config I know works but it been hard hunting down settings. the Changelog md file is a good place to fish for them.

**Reply by pjanzen** [reply]
In the end I rumbled trough the source code and found a properties file with allot of config options. I am not sure of those are all of them. A page with al the config options and a short explanation of those options would help a lot I think.

**Reply by ZergRushJoe** [reply]
I agree

**Reply by apanicker-nflx** [reply]
Thanks for the feedback. We will publish an exhaustive list of configuration options to the docs page soon.

**Reply by carlosvaquiro** [reply]
Hi, is there a link to the configuration options docs?
